tex/artikel/O-Woche_Programm.tex

```diff
@@ -18,161 +18,286 @@
+	}
+    &
+% Fr
+	\multirow{2}[15]{0.8\fibprogrammcw}{\textbf{Infoveranstaltung~III}\fibnlx
+		(jDPG \& andere studentische Gruppierungen)\fibnlx[0.5em]
+		\textbf{Plenum \& Preisverleihung}\fibnlx
+		\hspace*{\fill}
+		\textit{Hörsaal~1}}
+		
 %% Horizontal Lines
-\\ \cline{1-1}\cline{3-3}\cline{5-5}
+\\ \cline{1-1}\cline{3-3}%\cline{6-6}
+
 %%
 %%  14:00
 %%
-14:00 \fibabstand &
+14:00 \fibabstand\fibabstand\fibabstand\fibabstand & 
 % Mo
-	&
-% Di
-	\multirow{3}{\fibprogrammcw}[4mm]{\textbf{Stadtspiel}\fibnlx
-    \textit{\GEINS, nur Präsenz}
-	}&
-% Mi
-	& 
+    & 
+% Di & Mi
+	\multirow{2}[2]{0.8\fibprogrammcw}{\textbf{Stadtspiel}\fibnl
+		\hspace*{\fill}
+		\textit{Treffen an der Freitreppe}} & & 
 % Do
-	\multirow{3}{\fibprogrammcw}[4mm]{\textbf{Stadtspiel}\fibnlx
-    \textit{\GZWEI, nur Präsenz}
-	}
+% 	\multirow{2}[2]{0.8\fibprogrammcw}{\textbf{Ersti-Begrüßung des Rektorats}\fibnl
+% 		\hspace*{\fill}
+% 		\textit{Leonardo-Campus} 
+% 	} 
+    &
+% 	\multirow{2}[8]{0.8\fibprogrammcw}{%
+% 		\textbf{"Kaffeetrinken" mit den Professoren}\fibnl
+% 		\hspace*{\fill}
+% 		\textit{Foyer IG1}
+% 	}
 %% Horizontal Lines
-\\ \cline{1-2}\cline{4-4}
+\\ \cline{1-2}\cline{5-6}
 %%
 %%  15:00
 %%
 15:00 \fibabstand &
-% 
-% Mo & Di & Mi & Do
- & & &
-\\ \cline{1-2}\cline{4-4}
+% Mo
+    \multirow{4}[8]{0.8\fibprogrammcw}{\textbf{Campusspiel}\fibnl
+		\hspace*{\fill}
+		\textit{Treffen an der Freitreppe}\fibnlx\fibnlx\fibnlx
+		anschließend Grillen vor der Fachschaft\fibnlx
+		(bei passender Wetterlage)}
+	& 
+% Di - Fr	
+	& & 
+	
+	\multirow{2}[2]{0.8\fibprogrammcw}{%
+		\textbf{Konstruktionswettbewerb}\fibnl
+		\hspace*{\fill}
+		\textit{Treffen im Foyer IG1}
+	}
+	&
+	\textbf{Restegrillen}
+\\ \cline{1-1}\cline{4-4}
 %%
-%%  ab 19:00
+%%  16:00
 %%
-ab 19:00 \fibabstand &
-% Mo
-    \textbf{Online-Party}\fibnlx
-    \textit{\GEINS, \GZWEI}
-    &
-% Di
-    &
+16:00 \fibabstand &
+% Mo 
+	 & 
+% Di		
+		& 
+		
 % Mi
-    \textbf{Online-Kneipentour}\fibnlx
-    \textit{\GEINS, \GZWEI}&
+	\multirow{2}[6]{0.8\fibprogrammcw}{%
+		\textbf{"Kaffeetrinken" mit den Professoren}\fibnl
+		\hspace*{\fill}
+		\textit{Foyer IG1}
+	}&
+% Do 
+% 	\multirow{2}[2]{0.8\fibprogrammcw}{%
+% 		\textbf{Konstruktionswettbewerb}\fibnl
+% 		\hspace*{\fill}
+% 		\textit{Treffen im Foyer IG1}
+% 	}
+    &
+% Fr 
+%% Horizontal Lines
+\\ \cline{1-1}
+%%
+%%  17:00
+%%
+17:00 \fibabstand & & & & &
+%% Horizontal Lines
+\\ \cline{1-1}\cline{4-5}
+%%
+%%  18:00
+%% Mo
+18:00 \fibabstand &	&
+% Di 
+% 	\multirow{2}{0.8\fibprogrammcw}{%
+% 		\textbf{Grillen}\fibnlx
+% 		(bei passender Wetterlage)
+% 	} 
+    & 
+
+% Mi	
+	&
 % Do
-\\ \hline
+	\multirow{3}{0.8\fibprogrammcw}{%
+		\textbf{Spieleabend}\fibnlx
+		Brettspiele, Kartenspiele, Pen~\&~Paper und mehr!\fibnl
+		\hspace*{\fill}
+		\textit{Seminarraum KP~104}
+	} &
+% Fr
+
+\\ \cline{1-1}\cline{3-3}
+%%
+%%  19:00
+%%
+19:00 \fibabstand &	&
+% Di 
+\textbf{Kneipenabend}\fibnlx
+		\hspace*{\fill}
+		\textit{im "Buddenturm"}
```

Comment:
Die Kneipe hatte Christoph gestern gesagt oder? Ich hab leider in dem Moment nicht zugehört. Auf jeden Fall steht die noch nicht auf der Website.

Reply:
Ja hat er - im ,,La Palma'' findet es statt